.github/workflows/sync-upstream-push.yml

```diff
@@ -0,0 +1,33 @@
+name: Sync with Upstream (push)
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    if: github.event.review.state == 'approved' && startsWith(github.event.pull_request.head.ref, 'sync-upstream-')
```

Comment:
Same as that last comment. Maybe we should require `@bot merge` or something like that. Someone might approve my conflict resolution while I'm in bed but I'd expect to handle it myself when I'm ready. Especially important later if this kicks off some CD!

Reply:
Sounds good! I like the idea of having commands.

Reply:
I worked on the commands, check this PR to see how they worked:
https://github.com/grafana/runner-images/pull/103

Reply:
Check https://github.com/grafana/runner-images/pull/104 to see the explanation I left for the commands.

I think you might have some to help with wording around the PR message, or about the commands, since we use `atlantis apply` I kept it simple and used `bot change` and `bot merge`.